golang/scripts/json_data/3_filled_requests.json

```diff
@@ -0,0 +1,7 @@
+[
+    {
+        "inputIndex": 0,
+        "outputIndex": 0,
+        "id": "01"
```

Comment:
do I represent `[]byte{1}` in json????

Reply:
errors include
"01"
```
ERROR: Expected 8 bytes, got 1 bytes
```
"0000001"
```
"ERROR: encoding/hex: odd length hex string"
```

Reply:
There is an implementation of marshall and unmarshall over here:

https://github.com/summa-tx/relays/blob/golang/golang/x/relay/types/requests.go#L66-L87

These should be represented as hex strings encoding 8 bytes.  So 1 is encoded as: `"0x0000000000000001"`